hack/windows/Start.ps1

```diff
@@ -0,0 +1,84 @@
+Param(
```

Comment:
should we have an option to only start Antrea but not kube-proxy in this script? In case the user is managing kube-proxy differently?

Reply:
Done.

Reply:
It was more of an open-ended question from me :) If we do have this parameter though, the `Stop.ps1` script should not unconditionally stop kube-proxy, it case it was started by the user directly and not by `Start.ps1`.

Reply:
Thanks Antonin, we indeed should consider this senario. So how do you think if we add `-StopKubeProxy` in `Stop.ps1` and set the value as 'false' by default?

Reply:
I think you can either add this parameter to `Stop.ps1` or consider auto-generating the correct `Stop.ps1` script every time `Start.ps1` is run.